tests_e2e/orchestrator/lib/agent_test_suite_combinator.py

```diff
@@ -192,13 +212,40 @@ def create_environment(c_env_name: str) -> Dict[str, Any]:
                         "c_env_name": c_env_name,
                         "c_marketplace_image_information_location": self._MARKETPLACE_IMAGE_INFORMATION_LOCATIONS[self.runbook.cloud],
                         "c_shared_resource_group_location": self._SHARED_RESOURCE_GROUP_LOCATIONS[self.runbook.cloud],
-                        "c_vm_tags": c_vm_tags
+                        "c_vm_tags": c_vm_tags,
+                        "c_deploy": True
                     }
+                
+                def create_environment_ext_sequencing(c_env_name: str) -> Dict[str, Any]:
+                    log.info("Creating VMSS for ExtSequencing scenario")
+                    curr_datetime = datetime.datetime.now().strftime("%Y%m%d-%H%M%S")
+                    rg_name = f"lisa-WALinuxAgent-extseq-{curr_datetime}-e{self._created_rg_count}"
```

Comment:
we should use the 'name' variable from the runbook instead of hardcoding "WALinuxAgent"